core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -474,6 +474,24 @@ private SqlLibraryOperators() {
       OperandTypes.family(SqlTypeFamily.STRING, SqlTypeFamily.TIME),
       SqlFunctionCategory.TIMEDATE);
 
+  @LibraryOperator(libraries = {BIG_QUERY})
+  public static final SqlFunction TIME_ADD = new SqlFunction("TIME_ADD",
+      SqlKind.OTHER_FUNCTION,
+      ReturnTypes.TIME, null, OperandTypes.ANY_ANY,
+      SqlFunctionCategory.TIMEDATE);
+
+  @LibraryOperator(libraries = {BIG_QUERY})
+  public static final SqlFunction INTERVAL_SECONDS = new SqlFunction("interval_second",
+        SqlKind.OTHER_FUNCTION,
+        ReturnTypes.INTEGER, null,
+        OperandTypes.ANY, SqlFunctionCategory.TIMEDATE) {
+
+      @Override public void unparse(SqlWriter writer, SqlCall call, int leftPrec, int rightPrec) {
```

Comment:
unparse should not be part of SqlLibraryOperators. It should go into dialect files like BigQuerySqlDialect.

Reply:
not required as there are many function which have unparse function in the method declaration and i also tried to do it in BigQuerySqlDailect but in dailect the switch cases are based on call.getKind() in which our function condition are not satisfied because these is SqlFunction not SQL KIND so thats why i have kept the unparse method of interval seconds in SQLLibraryOperator

Reply:
as discussed please remove unparse method from here and add it in specific dialect.

Reply:
resolved